llvmlite/tests/test_binding.py

```diff
@@ -356,7 +356,7 @@ def test_set_option(self):
     def test_version(self):
         major, minor, patch = llvm.llvm_version_info
         # one of these can be valid
-        valid = [(9, 0), (8, 0), (7, 0), (7, 1)]
+        valid = [(10, 0), (9, 0), (8, 0), (7, 0), (7, 1)]
```

Comment:
Should 8 and 7 be in here still?

Reply:
I'm not sure, if we don't support those LLVM versions for the next release, then perhaps not?

Reply:
i think we can stop testing these

Reply:
Fixed.

Reply:
this is fixed